django/test/runner.py

```diff
@@ -161,7 +161,12 @@ def addFailure(self, test, err):
         self.stop_if_failfast()
 
     def addSubTest(self, test, subtest, err):
-        raise NotImplementedError("subtests aren't supported at this time")
+        # We follow Python 3.5's implementation of unittest.TestResult.addSubTest()
+        # by not doing anything when a subtest is successful.
+        if err is not None:
+            self.check_picklable(test, err)
```

Comment:
It looks like you should check the picklablility of `subtest`, since it's what you're adding to the event stream in the next line.